src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.Tests.csproj

```diff
@@ -3,8 +3,6 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{F7D9984B-02EB-4573-84EF-00FFFBFB872C}</ProjectGuid>
```

Comment:
As discussed please verify this fixes the hang you repro when these two flags are switched on.